packages/openid4vc-client/src/OpenId4VcClientApi.ts

```diff
@@ -32,4 +33,16 @@ export class OpenId4VcClientApi {
       checkRevocationState: checkRevocationState,
     })
   }
+
+  public async generateCodeVerifier(): Promise<string> {
```

Comment:
This method gives me the impression I can make a code for a verifier. What's purpose of this method?